### PR TITLE
Credentials acceptance tests updated.

### PR DESCRIPTION
**Pull request by tasawernawaz**
Credentials acceptance test are updated according to UI changes in LMS dashboard. 
This PR will be merged after this [PR](https://github.com/edx/edx-platform/pull/12468) (Containing changes in learner's dashboard) in edX platform.

Please review this @awais786 @waheedahmed @zubair-arbi @ahsan-ul-haq

### CODE REVIEW

File: acceptance_tests/test_rendering.py [hunk 1]
```diff
@@ -41,4 +47,10 @@ def test_student_dashboard_with_revoked_certificate(self):
         student_dashboard = LMSDashboardPage(self.browser).wait_for_page()
         student_dashboard.is_browser_on_page()
 
-        self.assertFalse(student_dashboard.are_credential_links_present())
+        # click on programs tab
```
Comment (zubair-arbi) [inline acceptance_tests/test_rendering.py]:
```
# click on "Programs" tab
```

**Comment by tasawernawaz** [comment]
@zubair-arbi Made suggested change. Please review.

**Comment by tasawernawaz** [comment]
@rlucioni Please review this.

File: acceptance_tests/pages.py [hunk 2]
```diff
@@ -70,11 +69,27 @@ def url(self):
     def is_browser_on_page(self):
         return self.browser.title.startswith('Dashboard')
 
+    def click_programs_tab(self):
```
Comment (zubair-arbi) [inline acceptance_tests/pages.py]: rename the name of this method to `go_to_programs_tab`

File: acceptance_tests/test_rendering.py [hunk 3]
```diff
@@ -20,11 +20,17 @@ def test_student_dashboard_with_certificate(self):
         student_dashboard = LMSDashboardPage(self.browser).wait_for_page()
         student_dashboard.is_browser_on_page()
 
-        self.assertTrue(student_dashboard.are_credential_links_present())
+        # click on "Programs" tab
+        student_dashboard.click_programs_tab()
 
-        credential_link = student_dashboard.get_credential_link()
+        student_dashboard_programs = LMSProgramsPage(self.browser).wait_for_page()
```
Comment (zubair-arbi) [inline acceptance_tests/test_rendering.py]: I think it make more sense to name it to `student_dashboard_programs_tab`

**Comment by tasawernawaz** [comment]
@zubair-arbi Made suggested changes. Please review.

**Comment by zubair-arbi** [comment]
LGTM 👍

File: acceptance_tests/pages.py [hunk 4]
```diff
@@ -70,11 +69,27 @@ def url(self):
     def is_browser_on_page(self):
         return self.browser.title.startswith('Dashboard')
 
+    def go_to_programs_tab(self):
+        self.q(css='a[href^="/dashboard/programs/"]').click()
+
+
+class LMSProgramsPage(LMSPage):
```
Comment (rlucioni) [inline acceptance_tests/pages.py]: Nitpick: to stay consistent with names on the LMS, I'd name this `LMSProgramListingPage`.

File: acceptance_tests/test_rendering.py [hunk 5]
```diff
@@ -20,11 +20,16 @@ def test_student_dashboard_with_certificate(self):
         student_dashboard = LMSDashboardPage(self.browser).wait_for_page()
         student_dashboard.is_browser_on_page()
 
-        self.assertTrue(student_dashboard.are_credential_links_present())
+        # go to "Programs" tab
```
Comment (rlucioni) [inline acceptance_tests/test_rendering.py]: This comment isn't necessary. Please remove it.

File: acceptance_tests/test_rendering.py [hunk 6]
```diff
@@ -41,4 +46,9 @@ def test_student_dashboard_with_revoked_certificate(self):
         student_dashboard = LMSDashboardPage(self.browser).wait_for_page()
         student_dashboard.is_browser_on_page()
 
-        self.assertFalse(student_dashboard.are_credential_links_present())
+        # go to "Programs" tab
```
Comment (rlucioni) [inline acceptance_tests/test_rendering.py]: This comment isn't necessary. Please remove it.

File: acceptance_tests/test_rendering.py [hunk 7]
```diff
@@ -20,11 +20,16 @@ def test_student_dashboard_with_certificate(self):
         student_dashboard = LMSDashboardPage(self.browser).wait_for_page()
         student_dashboard.is_browser_on_page()
 
-        self.assertTrue(student_dashboard.are_credential_links_present())
+        # go to "Programs" tab
+        student_dashboard.go_to_programs_tab()
+        student_dashboard_programs_tab = LMSProgramsPage(self.browser).wait_for_page()
```
Comment (rlucioni) [inline acceptance_tests/test_rendering.py]: Again, to stay consistent with variable names on the LMS, I'd name this `program_listing_page`.

**Comment by rlucioni** [comment]
Just a few minor comments inline, 👍 otherwise.

**Pull request merged**